src/styles/fonts.scss

```
@@ -0,0 +1,6 @@
+* {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+}
```

```suggestion
html {
  font-family: "Roboto", sans-serif;
  font-style: normal;
  font-weight: 400;
  font-size: 12px;
}
```
good place for setting genereal font styles is html selector